ctru-rs/examples/gfx-3d-mode.rs

```diff
@@ -0,0 +1,72 @@
+use ctru::gfx::TopScreen3D;
+use ctru::prelude::*;
+
+/// See `graphics-bitmap.rs` for details on how the image is generated.
+///
+/// WARNING: this example uses 3D mode in a rather unnatural way, and should
+/// probably not be viewed for too long or at all if you are photosensitive.
+
+const IMAGE: &[u8] = include_bytes!("assets/ferris.rgb");
+const ZERO: &[u8] = &[0; IMAGE.len()];
```

Comment:
These should probably be `static` not `const`.

Reply:
I could do that for `ZERO`, but `IMAGE` has to be `const` in order to use `IMAGE.len()` so I thought it would just be consistent that way. I can change it though